docs/modifiers/scroll-into-view.md

```diff
@@ -1,19 +1,41 @@
 # scroll-into-view
 
-This modifier calls `scrollIntoView` on the modified element.
+This modifier scrolls to the associated element. By default it uses `scrollIntoView`, but if an offset is passed as an option it uses `scrollTo` and calculates the `options.top` attribute.
 
 
 ## When you should use this modifier
 
-You should use this modifier whenever you need to have an element scrolled into view on element insert.
+You should use this modifier whenever you need to have an element scrolled into view. If there is a element, such as a fixed header, passing in an `offset` will scroll to the element minus the `offset` value.
+
 
 
 ## Basic Usage
 
 `scroll-into-view` expects the named `shouldScroll` parameter and an optional `options` named parameter. See [scrollIntoView](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView) for the list of possible values and properties of `options`.
 
-```handlebars{data-execute=false}
-<div {{scroll-into-view shouldScroll=this.shouldScrollPromise options=true}}></div>
+
+```handlebars
+  <div {{scroll-into-view shouldScroll=this.shouldScroll options=(hash behavior="smooth")}}>
+    <button type="button" {{on "click" this.onScrollIntoView}}>
+      Trigger scroll-into-view on click
+    </button>
+  </div>
+```
+
+`shouldScroll` can be either a Boolean or a Promise that resolves to a truthy or falsy value. It does not handle a rejected Promise.
+
+
+### Usage with offset
+
+When passing in an offset, it will call `scrollTo`. See [scrollTo](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollTo) for the list of possible values and properties of `options`.
```

Comment:
Let's document that only `left` and `behavior` are supported. You don't support passing in `top`, as `top` is calculated based on `offset`. So, let's also document that `offset` is the expected property here. It's best to be explicit rather than rely on people understanding the example below by itself.

Reply:
Let's also note that `offset` is measured in pixels (if I'm correct in that assumption)

Reply:
After offline discussion, moved to use `topOffset` and `leftOffset` to make this more generic and useful for horizontal scroll use-cases.

Both offsets are measured in pixels, since that's also what `scrollTo` expects, but added a comment to explicitly call that out.